nearcache/config.go

```diff
@@ -17,8 +17,10 @@
 package nearcache
 
 import (
+	"encoding/json"
 	"fmt"
 	"math"
+	"unsafe"
```

Comment:
Inaccurate line for the comment, can you reorder files on `nearcache.Config` for alignment

Reply:
I guess this comment belongs to another line?

Reply:
Yes sorry for that, there are field alignment issues on the Config struct in this file

Reply:
This is fixed, resolving